pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-parent</artifactId>
+		<version>1.2.0</version>
+	</parent>
+	<artifactId>bankrupt-officer-search-api</artifactId>
+	<version>unversioned</version>
+	<name>bankrupt-officer-search-api</name>
+	<description>CH API handling CRUD operations on Bankrupt Officer Search Service</description>
```

Comment:
Could you change this description please as it's not doing CRUD operations, it's only performing a read operation at the moment